udata_gouvfr/theme/templates/dataset/display.html

```diff
@@ -1,3 +1,12 @@
 {% extends 'dataset/display.html' %}
 
 {% set community_subtitle = _('Explore with %(certifier)s', certifier=config.SITE_TITLE ) %}
+
+{% block left_column_bottom %}
+<!-- dataset recommendations -->
+<script src="{{ theme_static('dataset-recommendations.js') }}" type="text/javascript" onload="udataDatasetRecos.load()"></script>
+<div id="dataset-recommendations" class="hidden-xs">
+    <h3>See also</h3>
```

Comment:
Translation is missing (ie. `<h3>{{ _('See also') }}</h3>`)

Reply:
👍